### ISSUE TITLE
Not whole item in node tree clickable

### ISSUE THREAD

**Issue by simonschaufi** [neos/neos-ui#2740]
### Description

![image](https://user-images.githubusercontent.com/941794/74543075-0bf27c00-4f45-11ea-81fd-db41ee10ed33.png)

![image](https://user-images.githubusercontent.com/941794/74543394-8fac6880-4f45-11ea-8e6c-abd5a6e99481.png)

As you can see I'm in the menu item for about 3px already but still I'm not able to click it. I have to move further up to be able to click it.

### Steps to Reproduce

1. Click menu item on Ubuntu

#### Expected behavior

hovering right into the highlighted container I should be able to click it

#### Actual behavior

I have to move about 5px into the container to be able to click it

### Affected Versions

Neos: 5

**Comment by Sebobo** [comment]
Can reproduce (with any browser)

**Comment by simonschaufi** [comment]
@Sebobo could you also fix it? ;)

**Comment by Sebobo** [comment]
It's not a simple fix, because the thing that keeps the cursor from allowing a click is the drag & drop target.

@dimaip can we somehow detect if a drag is going on, and if not hide all those drop targets?
We really don't need them if nothing is dragged.

**Comment by simonschaufi** [comment]
ah ok, thank you for your investigation

**Comment by dimaip** [comment]
@Sebobo theoretically we could, though we should make sure that it doesn't start jumping on drag start...
All I remember about it that the whole the was kind of tricky...

**Comment by simonschaufi** [comment]
Thank you for fixing this issue!
